### PR TITLE
Pull all the balances

### PR DESCRIPTION
**Pull request by toddkao**
lift most valuable asset to top

### CODE REVIEW

File: packages/widget-v2/src/state/balances.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  return {
+    queryKey: ["skipBalances", params],
+    queryFn: async () => {
+      if (!params) {
+        throw new Error("No balance request provided");
+      }
+
+      return skip.balances(params);
+    },
+    retry: 1,
```
Comment (codingki) [inline packages/widget-v2/src/state/balances.ts:21]: can we add refetchInterval, maybe 30s or 1min or what ever you think it make sense

File: packages/widget-v2/src/pages/SwapPage/SwapPage.tsx [hunk 2]
```diff
@@ -68,30 +68,30 @@ export const SwapPage = () => {
...
+          acc[asset.chainID].denoms.push(asset.denom);
+        } else {
+          acc[asset.chainID] = {
+            address: address,
+            denoms: [asset.denom],
+          };
+        }
+      }
+      return acc;
+    }, {} as Record<string, { address: string, denoms: string[] }>);
```
Comment (codingki) [inline packages/widget-v2/src/pages/SwapPage/SwapPage.tsx:89]: I think we can have this effect in `useSourceAccount`

**Review by codingki — APPROVED** [review]

File: packages/widget-v2/src/state/balances.ts (hunk 1 above)
Comment (toddkao) [reply packages/widget-v2/src/state/balances.ts:21]: I'll add 1 min for now, I'm a little worried about this endpoint xd

Comment (toddkao) [reply packages/widget-v2/src/state/balances.ts:22]: maybe we should consider only refetching balances for a specific asset after you actually do a swap

**Pull request merged**